src/main/java/core/basesyntax/dao/impl/MessageDaoImpl.java

```diff
@@ -12,21 +15,62 @@ public MessageDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Message create(Message entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(entity);
```

Comment:
Name  of var is better 'message'. session.persist(message). We have cascades: CascadeType.PERSIST, CascadeType.DELETE.

Reply:
Done